response_operations_ui/templates/collection_exercise/collection-exercise.html

```diff
@@ -25,7 +24,11 @@
 
         <div class="grid__col col-5@l push-1@l">
             {% include 'collection_exercise/ce-reference-information.html' %}
+            {% if survey.surveyMode == 'EQ' %}
```

Comment:
I really think we should feature flag this off

Reply:
feature flagging off in prod via frontstage now. Thanks Warren